_chapters/js/dependencies.md

```diff
@@ -22,6 +22,29 @@ It's not needed and should be phased out and replaced by npm. More information c
 
 npm installs its dependencies in the `node_modules` directory. Common conventions dictate that `node_modules` should be excluded from source control by adding it to your project's `.gitignore`, primarily because Node.js-friendly environments (such as 18F's deployment service, [Cloud Foundry], and other such as [Heroku]) recognize the existence of `package.json` and automatically install dependencies as needed.
 
+### Publishing
+#### Scoping a package to the 18F npm org
+18f has an [npm organization](https://www.npmjs.com/org/18f) that is meant to organize permissions and packages related to 18f. As an 18F developer, when publishing a package, you have the choice whether to scope a package to the 18F org or not. Scoped package will always be prefixed with `@18f/` before their package name and can have their permissions managed by people in the org. More information about scoped packages can be found on the [npm documentation](https://docs.npmjs.com/misc/scope).
```

Comment:
Should probably be consistent with capitalization in our name. I think "18F" is preferred.